template/en/default/global/setting-descs.none.tmpl

```diff
@@ -56,6 +56,7 @@
    "possible_duplicates"              => "Display possible duplicates when reporting a new $terms.bug",
    "requestee_cc"                     => "Automatically add me to the CC list of $terms.bugs I am requested to review",
    "api_key_only"                     => "Require API key authentication for API requests",
+   "use_elasticsearch"                => "Use elasticsearch for $terms.bug searches when possible",
```

Comment:
"when possible" is, perhaps, a bit of a fine point. Why is it opt-in for logged-in users rather than opt-out? How could a user discover that in any way other than the above code patch?